src/Layer.js

```diff
@@ -334,46 +337,77 @@ export default class Layer {
     }
 
     /**
-     * Change layer visibility to visible
-     *
-     * @memberof carto.Layer
-     * @instance
-     * @api
-     *
-     * @fires updated
+     * Custom Layer API: `onAdd` function
      */
-    show () {
-        this._visible = true;
-        this._integrator.changeVisibility(this);
-        this.requestData();
-        this._fire('updated');
+    onAdd (map, gl) {
```

Comment:
Can we change the name of this? Users may look at it and think that this is public and it is used to get an event when the map is loaded

Reply:
ah, true... yes, I think we should ask them